pcap.pxd

```diff
@@ -0,0 +1,164 @@
+#
+# pcap.pxd
+#
+# $Id$
+
+"""packet capture library
+
+This module provides a high level interface to packet capture systems.
+All packets on the network, even those destined for other hosts, are
+accessible through this mechanism.
+"""
+
+__author__ = 'Dug Song <dugsong@monkey.org>'
+__copyright__ = 'Copyright (c) 2004 Dug Song'
+__license__ = 'BSD license'
+__url__ = 'https://github.com/pynetwork/pypcap'
+__version__ = '1.2.0'
```

Comment:
Same here, can we have these be defined in only one of the two places? Otherwise it becomes a bit more painful , especially when bumping the version.